corehq/apps/dump_reload/sql/dump.py

```diff
@@ -115,7 +116,7 @@
     FilteredModelIteratorBuilder('sms.KeywordAction', SimpleFilter('keyword__domain')),
     FilteredModelIteratorBuilder('sms.QueuedSMS', SimpleFilter('domain')),
     FilteredModelIteratorBuilder('sms.SelfRegistrationInvitation', SimpleFilter('domain')),
-    FilteredModelIteratorBuilder('sms.SQLMobileBackend', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('sms.SQLMobileBackend', DomainOrNullFilter('domain')),
```

Comment:
Are these models for sure okay to dump?  Does this copy over keys and give them access to SMS backends that Dimagi is responsible for (or might get charged for)?

Another option would be to recreate these manually on their server afterwards.

Reply:
You are absolutely right. This is a terrible idea.

I just realised I can filter SQLMobileBackendMapping by the domain of its backend. That'll give us only the ones we can load. Any others will need to be added manually.